lithium/lithium.py

```diff
@@ -1145,12 +1187,14 @@ def error(subself, message):  # pylint: disable=no-self-argument
         grp_atoms.add_argument(
             "-c", "--char",
             action="store_true",
-            help="Don't treat lines as atomic units; treat the file as a sequence of characters rather than a sequence of lines.")
+            help="Don't treat lines as atomic units; "
+            "treat the file as a sequence of characters rather than a sequence of lines.")
         grp_atoms.add_argument(
             "-s", "--symbol",
             action="store_true",
             help="Treat the file as a sequence of strings separated by tokens. "
-                 "The characters by which the strings are delimited are defined by the --cutBefore, and --cutAfter options.")
+                 "The characters by which the strings are delimited are defined by "
+                 "the --cutBefore, and --cutAfter options.")
```

Comment:
I know why it complained. Apparently if there is the ` +` at the end of the previous line, pylint wants it lined up with `help=` above, but since we are removing the trailing ` +`, it no longer is an issue. :)